baselines/teams.md

```diff
@@ -498,12 +543,14 @@ Docs](https://docs.microsoft.com/en-us/microsoftteams/policy-assignment-overview
 
 ### Implementation
 
-To configure the Meeting policies for cloud video recording:
+All the settings in this section are configured in the **Microsoft Teams admin center**.
```

Comment:
For similar statements made for MS.TEAMS.1 through MS.TEAMS.5, "Microsoft Teams admin center" was a link. Recommend changing this to a link for consistency.

Reply:
Updated to add a link.